packages/kmod-5.15-nvidia/kmod-5.15-nvidia.spec

```diff
@@ -91,6 +109,24 @@ install -d %{buildroot}%{_cross_tmpfilesdir}
 install -d %{buildroot}%{_cross_unitdir}
 install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/{drivers,ld.so.conf.d}
 
+install -p -m 0644 %{S:203} %{buildroot}%{_cross_unitdir}
+
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/nvidia
+install -p -m 0644 %{S:204} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/nvidia/fabricmanager.cfg
+
+# Begin NVIDIA fabric manager
+pushd fabricmanager-linux-%{fm_arch}-%{tesla_ver}-archive
+install -d %{buildroot}%{_cross_bindir}
+install -p -m 0755 bin/nv-fabricmanager %{buildroot}%{_cross_bindir}/nvidia-fabricmanager
```

Comment:
Does renaming this binary affect anything looking for `nv-fabricmanager` vs `nvidia-fabricmanager`?

Reply:
I have changed it back to nv-fabricmanager to keep is consistent with other binaries. good call out. :)